sbor-derive-common/src/utils.rs

```diff
@@ -192,16 +192,7 @@ pub fn get_hash_of_code(input: &TokenStream) -> [u8; 20] {
 }
 
 pub fn get_unique_types<'a>(types: &[&'a syn::Type]) -> Vec<&'a syn::Type> {
-    let mut seen = HashSet::<&syn::Type>::new();
-    let mut out = Vec::<&syn::Type>::new();
-    for t in types {
-        if seen.contains(t) {
-            continue;
-        }
-        seen.insert(t);
-        out.push(t);
-    }
-    out
+    types.into_iter().unique().cloned().collect()
```

Comment:
_Note from author:_

I hope itertools is not a problematic dep (we do use it in other places already).
The `.unique()` maintains the original order (which is what we care about, I guess).